Tutorials/CNTK_201C_CIFAR-10_ResNet20_Learner_Comparison.ipynb

```diff
@@ -0,0 +1,363 @@
+    "        x = range(len(loss))\n",
+    "        plt.plot(x, loss, color=colors[i])\n",
+    "    plt.savefig(img_file)\n",
+    "    # plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The whole main function looks as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "if __name__ == '__main__':\n",
+    "    log_files = ['momentum_sgd.log']\n",
+    "    losses = [None]*len(log_files)\n",
+    "    metrics = [None]*len(log_files)\n",
+    "    times = [None]*len(log_files)\n",
+    "    for i, log_file in enumerate(log_files):\n",
+    "        losses[i], metrics[i], times[i] = parse_file('../log/' + log_file)\n",
+    "    \n",
+    "    plot(losses, '../img/losses.png')\n",
+    "    plot(metrics, '../img/metric.png')\n",
+    "    plot(times, '../img/times.png')                "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "So far, we have three `png` pictures under the `./img/` folder, which are generated from the `log` files under `./log/` folder."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Change the learner in example code"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It is easy to change the learner in the example code given to a exsiting learner, you just need to first include new learner:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from cntk.learners import sgd, momentum_sgd, learning_rate_schedule, momentum_as_time_constant_schedule, UnitType"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "then change the parameters in the following single line of code:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# learner = momentum_sgd(z.parameters, lr_schedule, mm_schedule,\n",
+    "#                        l2_regularization_weight = l2_reg_weight)\n",
+    "learner = sgd(z.parameters, lr_schedule, \n",
+    "              l2_regularization_weight = l2_reg_weight)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## User Defined Learner"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you want to use some learner that are not currently implemented in the library, you can just define it yourself. We take AdaDelta [3] as an example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from cntk.learners import UserLearner\n",
+    "\n",
+    "class AdaDelta(UserLearner):\n",
```

Comment:
I would like this to move under CNTK/Manual. Make the title Manual_how_to_use_and_evaluate_different_learners.ipynb.

Please also add a test for this notebook under CNTK/Tests/EndToEndTests/CNTKv2Python/Manual